ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -231,12 +230,7 @@ Go To RHODS Dashboard
   Wait for RHODS Dashboard to Load
 
 Load Expected Data Of RHODS Explore Section
-    ${version_check}=   Is RHODS Version Greater Or Equal Than  1.19.0
-    IF  ${version_check}==True
-        ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH_LATEST}
-    ELSE
-        ${apps_dict_obj}=  Load Json File  ${APPS_DICT_PATH}
```

Comment:
the old file (i.e. APPS_DICT_PATH) was containing mentions to RHOSAK too. Since the version check is not used anymore, I deleted the check and the file